content/setup/frameworkPackages/es6Support.md

```diff
@@ -49,8 +49,8 @@ As the ES6 version is composed of many separated files the usual way to consume
 2. Open **GitBash** (or similar) for **MyAwesomeApp**
 3. Run `npm init`
 4. Run `npm install webpack webpack-cli webpack-dev-server --save-dev`
-5. Run `npm install @babylonjs/core`
-6. Create **index.html** file in **MyAwesomeApp** as described in `First App Section` below
+5. Run npm install @babylonjs/core and npm install --save-dev @babylonjs/materials

```

Comment:
Removing the code section will make it confusing. This would be better:

```suggestion
5. Run `npm install @babylonjs/core @babylonjs/materials`
```

Reply:
It's important to include the step that involves creating the index.html file inside the public folder. I've seen two topics created by beginners on the forum for the same issue. Example: [https://forum.babylonjs.com/t/i-am-unable-to-open-babylon-locally-in-the-browser/42123 ](url).

Reply:
I never said this shouldn't happen. all I said is that you removed the code definition. the ` characters. which will make it seem like part of the test, instead of copy-able command.